ssd_bandwidth/configuration.yml

```diff
@@ -0,0 +1,44 @@
+---
+configuration:
+    name: SSD Bandwidth
+    description: Benchmark SSD IOPS using FIO
+    directory: ssd_bandwidth
+    cron: "0 * * * *"
+    instance-number: 1
+    instance-types:
+      - i4g.4xlarge
+
+nodes:
+  - node-id: 0
+    ansible-configuration: ansible.yml
```

Comment:
It is correct, but you use only 1 node, so there is no point in using default and concrete configuration.
You can merge it by moving this line to `node-1` to make it more clear and remove `node-0`